### ISSUE TITLE
Bug: Remove alt text for StarSearch logos

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#3264]
Alt text is a good thing, but in case where an icon is strictly ornamental it should have empty alt text. As well, you generally do not need to say it's a logo or image.

This is the line to change.

https://github.com/open-sauced/app/blob/41ee9f290f97031aec09b6cb7afe3e388808b386/components/organisms/TopNav/top-nav.tsx#L89

```diff
-            <img src="/assets/star-search-logo.svg" alt="Star Search logo" className="w-5 h-5" />
+            <img src="/assets/star-search-logo.svg" alt="" className="w-5 h-5" /> 
```

**Comment by nickytonline** [comment]
This is a good first issue, so if you've already contributed to the project, leave this one for the new comers to the project.

**Comment by nikkhielseath** [comment]
.take
